docs/sphinx/user/configuring_simeng.rst

```diff
@@ -21,7 +23,7 @@ SimEng cores can be one of three types:
 
 These core types are primarily referred to as core "archetypes".
 
-.. Note:: Currently, the configuration files do not take into account the core archetype being modelled. However, future developments plan for the exemption of those options not used under the selected core archetype. For example, reservation station definitions under the ``inorderpiplined`` archetype will not be required.
+.. Note:: Currently, the configuration files do not take into account the core archetype being modelled and require all parameters to be defined, even if unused (e.g. reservation station definitions for an ``emulation`` core archetype). However, future developments plan for the exemption of those options not used under the selected core archetype.
```

Comment:
This statement is not entirely true as some parameters do have default values. Perhaps not vital that we change this note as it is "safer" if all config options are defined, but worth pointing nonetheless.

Reply:
I'll change it to include this, no harm